.github/ISSUE_TEMPLATE/offboarding.md

```diff
@@ -72,33 +88,47 @@ ticket has been completed. At that time, put the ticket back to triage.
+- [ ] Assigned themselves the `GitLab own` role on the `prod` instance of GitLab <sub>or the offboarded employee/contractor did not perform the `system administrator` role</sub>
+- [ ] Assigned themselves the `GitLab own` role on the `anvildev` instance of GitLab <sub>or the offboarded employee/contractor did not perform the `system administrator` role</sub>
+- [ ] Assigned themselves the `GitLab own` role on the `anvilprod` instance of GitLab <sub>or the offboarded employee/contractor did not perform the `system administrator` role</sub>
+- [ ] Removed the offboarded employee/contractor from the list of members on the Group members page in the `dev` instance of GitLab
+- [ ] Removed the offboarded employee/contractor from the list of members on the Group members page in the `prod` instance of GitLab
+- [ ] Removed the offboarded employee/contractor from the list of members on the Group members page in the `anvildev` instance of GitLab
+- [ ] Removed the offboarded employee/contractor from the list of members on the Group members page in the `anvilprod` instance of GitLab
+- [ ] Set the offboarded employee/contractor's account status to `Blocked` on the Users page in the Admin Area of the `dev` instance of GitLab
+- [ ] Set the offboarded employee/contractor's account status to `Blocked` on the Users page in the Admin Area of the `prod` instance of GitLab
+- [ ] Set the offboarded employee/contractor's account status to `Blocked` on the Users page in the Admin Area of the `anvildev` instance of GitLab
+- [ ] Set the offboarded employee/contractor's account status to `Blocked` on the Users page in the Admin Area of the `anvilprod` instance of GitLab
 - [ ] Removed all role assignments from the offboarded employee/contractor in the `dev` instance of GitLab
 - [ ] Removed all role assignments from the offboarded employee/contractor in the `prod` instance of GitLab
 - [ ] Removed all role assignments from the offboarded employee/contractor in the `anvildev` instance of GitLab
 - [ ] Removed all role assignments from the offboarded employee/contractor in the `anvilprod` instance of GitLab
-- [ ] Blocked the GitLab account owned by the offboarded employee/contractor on every GitLab instance in the system
-- [ ] Removed the Google Service accounts owned by the offboarded employee/contractor in the `platform-hca-dev` Google Cloud project
-- [ ] Removed the Google Service accounts owned by the offboarded employee/contractor in the `platform-hca-prod` Google Cloud project
-- [ ] Removed the Google Service accounts owned by the offboarded employee/contractor in the `platform-anvil-dev` Google Cloud project
-- [ ] Removed the Google Service accounts owned by the offboarded employee/contractor in the `platform-anvil-prod` Google Cloud project
-- [ ] Performed an account review and disabled any accounts owned by the offboarded employee/contractor
-- [ ] Performed a resource inventory and deleted any resources owned by the offboarded employee/contractor
```

Comment:
Same for this item.

Reply:
> What checklist items cover "The replacement changes the credentials to all AWS root user accounts" for organization admin?

I can't find a record of the decision, however it is my understanding that the offboarding and replacement of an Organization administrator is outside the scope of this checklist to offboard a person from team Boardwalk.

Reply:
> the offboarding (and replacement) of an Organization administrator is outside the scope of this checklist

that's my understanding, too